protos/dcs/unit/unit.proto

```diff
@@ -39,6 +44,44 @@ message GetPositionResponse {
   dcs.common.Position position = 1;
 }
 
+message ExportRequest {
+  string name = 1;
+}
+
+/**
+ * The orientation of an object in 3D space.
+ */
+message Orientation {
+  // The normalized direction the object is pointing to.
+  dcs.common.Vector forward = 1;
+  // The normalized direction the three line (right wing) is pointing to.
+  dcs.common.Vector right = 2;
+  // The normalized up vector (orthogonal to forward and right).
+  dcs.common.Vector up = 3;
+  // Yaw in degrees - clockwise relative to the true north.
+  double yaw = 4;
+  // Pitch in degrees - positive when taking-off.
+  double pitch = 5;
+  // Roll in degrees - positive when rolling the aircraft to the right.
+  double roll = 6;
+}
+
+message ExportResponse {
+  dcs.common.Position position = 1;
+  // Distance between DCS' map origin to object in meters on west-east axis.
+  double u = 2;
+  // Distance between DCS' map origin to object in meters on north-south axis.
+  double v = 3;
+  // The heading of the object on a flat world. To get the heading corrected by the projection error
+  // when going from a flat to a spherical world, use `orientation.yaw` instead.
+  double heading = 4;
+  Orientation orientation = 5;
+  // The direction the object is traveling to, and speed (magnitude of the vector) the object is
+  // traveling with.
+  dcs.common.Vector velocity = 6;
+  double time = 7; // Time in seconds since the scenario started.
+}
```

Comment:
yaw, pitch, roll, u, v and heading map what TacView is using (both in naming and semantic)